### PR TITLE
chore: correct typos docs

### PR DESCRIPTION
**Pull request by sky-coderay**
**What changed? Why?**
Minor typos fixed in docs, significant repetition adjusted in DesktopNav.tsx

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

**Comment by sky-coderay** [comment]
@cb-heimdall finish

**Comment by sky-coderay** [comment]
@brendan-defi finish

File: apps/bridge/src/components/Nav/DesktopNav.tsx [hunk 1]
```diff
@@ -119,7 +119,7 @@ function IconLink({
 
 function DesktopNav({ color }: DesktopNavProps) {
   return (
-    <div className="hidden h-full w-fit flex-grow flex-row items-center justify-between lg:flex">
```
Comment (brendan-defi) [inline apps/bridge/src/components/Nav/DesktopNav.tsx:122]: @dschlabach given that it's the default, no issues with removing flex-row here, right?

Comment (dschlabach) [reply apps/bridge/src/components/Nav/DesktopNav.tsx:122]: no issue

**Review by brendan-defi — APPROVED** [review]

**Review by brendan-defi — APPROVED** [review]

**Comment by brendan-defi** [comment]
hi @sky-coderay thanks for your contribution! we've approved and merged 🙇

**Pull request merged**